lib/KettleRequest.js

```diff
@@ -251,8 +252,8 @@ kettle.request.http.errorHandler = function (res, error) {
 
 /**
  * Send a successful payload to the client if the success event is fired.
- * @param  {Object} res an Express response object.
- * @param  {Object} response a success payload.
+ * @param {Component} request - A kettle.request component
+ * @param {Object} response a success payload.
```

Comment:
For consistency, probably a dash is warranted here (and the "a" should be capitalised).